rma_account_unreconciled/models/rma_line.py

```diff
@@ -63,13 +63,28 @@ def _get_purchase_unreconciled_base_domain(self):
     def action_view_unreconciled(self):
         self.ensure_one()
         acc_item = self.env["account.move.line"]
-        domain = self._get_purchase_unreconciled_base_domain()
+        domain = self._get_rma_unreconciled_base_domain()
         unreconciled_domain = expression.AND([domain, [("rma_line_id", "=", self.id)]])
         unreconciled_items = acc_item.search(unreconciled_domain)
         action = self.env.ref("account.action_account_moves_all")
         action_dict = action.read()[0]
         action_dict["domain"] = [("id", "in", unreconciled_items.ids)]
         return action_dict
 
-    def action_reconcile(self):
-        return True
+    def action_open_reconcile(self):
+        aml_model = self.env["account.move.line"]
+        action = self.action_view_unreconciled()
+        amls = aml_model.search(action.get("domain", [(1, "!=", 1)]))
```

Comment:
Why [(1, "!=", 1)]?

Reply:
Is to prevent search all account.moves, but I changed condition